tests/testthat/test-retrieveData.R

```diff
@@ -2,6 +2,8 @@ globalassign <- function(...) {
   for (x in c(...)) assign(x, eval.parent(parse(text = x)), .GlobalEnv)
 }
 
+tempDir <- tempdir()
```

Comment:
`tempdir()` gibt in der selben session immer den gleichen Pfad zurück, die extra Variable hier ist also nicht unbedingt notwendig. Allgemein hatte ich mit `tempdir()` schon öfter Probleme beim zweiten Ausführen der Tests in der selben session, denn dann waren die Dateien vom ersten Durchgang noch im tempdir und manche Tests schlugen fehl. Deshalb würde ich eher `withr::local_tempdir()` empfehlen, das gibt immer einen leeren Pfad zurück der automatisch am Ende der Funktion gelöscht wird.